functions/webhooks/line/FlexToLine.protected.ts

```diff
@@ -69,9 +69,10 @@ const sendLineMessage =
     });
 
     return {
-      status: response.status,
+      ok: response.ok,
+      resultCode: response.status,
       body: await response.json(),
-      headers: Object.fromEntries(Object.entries(response.headers)),
+      meta: Object.fromEntries(Object.entries(response.headers)),
```

Comment:
Same comment about changes in this file

Reply:
I standardised the return types from the 'send message' implementations for conversations custom channels which meant we could use generic error handling code in the common function rather than having to handle it separately for each channel - this necessitated some small changes to the Line conversations implementation to align with the new return type